PJ3T3_Postie/Core/Setting/View/InformationView.swift

```diff
@@ -0,0 +1,241 @@
+//
+//  InformationView.swift
+//  PJ3T3_Postie
+//
+//  Created by 권운기 on 2/11/24.
+//
+
+import SwiftUI
+
+struct InformationView: View {
+    @Binding var isThemeGroupButton: Int
+    
+    var body: some View {
+        let postieColors = ThemeManager.themeColors[isThemeGroupButton]
+        var appVersion: String {
+                if let version = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String,
+                   let build = Bundle.main.object(forInfoDictionaryKey: "CFBundleVersion") as? String {
+                    return "\(version) (\(build))"
```

Comment:
```suggestion
                    return "\(version)"
```
이제 버전에 빌드도 반영하기로 했으니 버전만 보여도 될 것 같아요!